test/apps/iodemo/ucx_wrapper.cc

```diff
@@ -1077,3 +1062,10 @@ bool UcxConnection::process_request(const char *what,
         }
     }
 }
+
+void UcxConnection::invoke_callback(UcxCallback *&callback, ucs_status_t status)
```

Comment:
If I’m not mistaken, it could be just `UcxCallback &callback`, since the name of the function is already pointed to the function.
And also it could be:
```
UcxCallback cb = callback;

callback = NULL;
cb(status);
```

Reply:
`UcxCallback` is a functor class not a function pointer, so `UcxCallback *&callback` is a reference to pointer to class instance